include/oneapi/tbb/detail/_flow_graph_node_impl.h

```diff
@@ -351,8 +354,11 @@ class function_input_base : public receiver<Input>, no_assign {
     }
 
     //! allocates a task to apply a body
+#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
+    template <typename Metainfo>
+#endif
     graph_task* create_body_task( const input_type &input
-                                  __TBB_FLOW_GRAPH_METAINFO_ARG(const message_metainfo& metainfo))
+                                  __TBB_FLOW_GRAPH_METAINFO_ARG(Metainfo&& metainfo))
```

Comment:
At this point, does it make sense to write macro `#if #elif` for method definition? 😄

Reply:
Done:)